### DISCUSSION TITLE
Hidden or "missing" features

### DISCUSSION THREAD

**Discussion by maehw** [xtech/self-o-mat#7] [Ideas]
Are the following features implemented and hidden or "missing" (possible enhancements)?

1. Count down before the camera triggers: So that one has time after having pushed the button to get into position. (Edit: On the self-o-mat DIY kit a countdown seems to be shown via the LED ring. So wanting a countdown (currently) implies use of that hardware and setting `has_button: true`, right?)
2. Multi shot mode: Taking several photos (2/3/4) at once and displaying them all on the screen for a specific time afterwards.
3. Is there a gallery view to show old shots?

**Comment by ClemensElflein** [comment]
> 
> 
> Are the following features implemented and hidden or "missing" (possible enhancements)?
> 
>     1. Count down before the camera triggers: So that one has time after having pushed the button to get into position. (Edit: On the self-o-mat DIY kit a countdown seems to be shown via the LED ring. So wanting a countdown (currently) implies use of that hardware and setting `has_button: true`, right?)

Yes, the Countdown is currently only available with the hardware. The click to take a photo feature was only used for development.

> 
>     2. Multi shot mode: Taking several photos (2/3/4) at once and displaying them all on the screen for a specific time afterwards.

This is not implemented yet, I'll create an issue for this.

> 
>     3. Is there a gallery view to show old shots?

This is not implemented as well.

I'll add issues for these enhancements
